gui/src/3D/gizmos/slateGizmo.ts

```diff
@@ -236,7 +235,7 @@ export class SlateGizmo extends Gizmo {
 
         const dragStart = (event: { position: Vector3 }) => {
             if (this.attachedSlate && this.attachedMesh) {
-                dimensionsStart.copyFrom(this.attachedSlate.dimensions);
+                dimensionsStart.set(this.attachedSlate.dimensions.x, this.attachedSlate.dimensions.y, 0.001);

```

Comment:
maybe use Epsilon from math.constants instead of setting 0.001? If it makes sense, of course.

Reply:
I used 0.001 since it was what was used before, but honestly I think epsilon would be fine.

Side note, I noticed that if the depth is 0 then you can't interact with the contents at all (as if the ray hit passes right through the slate). Do you know anything about that?